docs/app/Examples/modules/Dropdown/Content/AsyncOptions.js

```diff
@@ -1,16 +1,19 @@
 import _ from 'lodash'
-import cx from 'classnames'
 import faker from 'faker'
 import React, { Component } from 'react'
 import { Button, Dropdown } from 'stardust'
 
-const getOptions = () => _.times(3, () => {
+const getOptions = () => _.times(5, () => {
```

Comment:
Needed the dropdown to scroll with options and have more items for search matches.  Again, this PR is not actually ready yet.